zookeeper-jute/src/main/java/org/apache/jute/compiler/JBuffer.java

```diff
@@ -79,7 +79,7 @@ public String genJavaEquals(String fname, String peer) {
     }
 
     public String genJavaHashCode(String fname) {
-        return "    ret = java.util.Arrays.toString(" + fname + ").hashCode();\n";
+        return "    ret = java.util.Arrays.hashCode(" + fname + ");\n";
     }
```

Comment:
- their will generate different `hashcode`?
- these two methods accept `array` parameter, not support the parameter `string`?

Reply:
* Yes.  Will generate different hashcodes.  Doesn't matter as hashcodes are not persisted anywhere (that I know of... that is just not a common thing to do).
* This method accepts `byte[]` which is the data type being used here.  The `fname` is just the name of the field, not the value being hashcode'ed